src/styles/utils/_functions.scss

```
@@ -0,0 +1,3 @@
+@function getCalendarWidth ($cell-size, $gap-size) {
+  @return $cell-size * 7 + $gap-size * 6+ $calendar-padding * 2;
+}
```

not good when function take variables from outside, if you change var name 'calendar-padding' this function will break
```suggestion
@function getCalendarWidth ($cell-size, $gap-size, $calendar-padding) {
  @return $cell-size * 7 + $gap-size * 6+ $calendar-padding * 2;
}
```